coffee-backend/src/main/kotlin/chat/letscoffee/security/oauth2/user/OAuth2UserInfo.kt

```diff
@@ -0,0 +1,10 @@
+package chat.letscoffee.security.oauth2.user
+
+abstract class OAuth2UserInfo(var attributes: Map<String?, Any?>?) {
```

Comment:
See comment in FacebookOAuth2UserInfo, where I thought this class was from a framework. This should be an interface, and the constructor should be defined in each implementing class. Preferably the constructor should be an object instead of a map, so that we have more checks on existence than in a regular map.